test/smoke/FirefoxPrivateVPNUITest/FirefoxPrivateVPNUITest/Common/UserCommonOperation.cs

```diff
@@ -81,7 +85,129 @@ public static IRestResponse AmIMullvad()
             request.AddHeader("Host", "am.i.mullvad.net");
             request.AddHeader("Cache-Control", "no-cache");
             request.AddHeader("Accept", "*/*");
-            return client.Execute(request);
+            return RetryExecute(client, request);
+        }
+
+        /// <summary>
+        /// Call Mullvad API to get current city.
+        /// </summary>
+        /// <returns>The API response.</returns>
+        public static IRestResponse GetCityViaMullvad()
+        {
+            var client = new RestClient("https://am.i.mullvad.net/city");
+            var request = new RestRequest(Method.GET);
+            request.AddHeader("cache-control", "no-cache");
+            request.AddHeader("Connection", "keep-alive");
+            request.AddHeader("Accept-Encoding", "gzip, deflate");
+            request.AddHeader("Host", "am.i.mullvad.net");
+            request.AddHeader("Cache-Control", "no-cache");
+            request.AddHeader("Accept", "*/*");
+            return RetryExecute(client, request);
+        }
+
+        /// <summary>
+        /// User click the toggle switch to connect to VPN.
+        /// </summary>
+        /// <param name="vpnClient">VPN session.</param>
+        /// <param name="desktop">Desktop session.</param>
+        public static void ConnectVPN(FirefoxPrivateVPNSession vpnClient, DesktopSession desktop)
+        {
+            // Verify user is not connected to Mullvad VPN
+            IRestResponse response = AmIMullvad();
+            Assert.AreEqual(HttpStatusCode.OK, response.StatusCode);
+            Assert.IsTrue(response.Content.Contains("You are not connected to Mullvad"));
+
+            // Click VPN switch toggle and turn on VPN
+            vpnClient.Session.SwitchTo();
+            MainScreen mainScreen = new MainScreen(vpnClient.Session);
+            mainScreen.ToggleVPNSwitch();
+            Assert.IsTrue(mainScreen.GetOnImage().Displayed);
+            Assert.IsFalse(mainScreen.GetOffImage().Displayed);
+
+            // Verify the windows notification
+            Thread.Sleep(TimeSpan.FromSeconds(2));
+            desktop.Session.SwitchTo();
+            WindowsNotificationScreen windowsNotificationScreen = new WindowsNotificationScreen(desktop.Session);
+            Assert.AreEqual("VPN is on", windowsNotificationScreen.GetTitleText());
```

Comment:
When testing strings within the application, should we test based on strings defined in our localized strings file instead of string literals?  Keep in mind these strings are subject to change in the future, and we also have support for other languages.